### ISSUE TITLE
"connection timed out" for Sanity Checker's 

### ISSUE THREAD

**Issue by charlesakalugwu** [openshift/openshift-azure#1363]
We are seeing some flakes wherein the CreateStatefulApp task of the e2e Sanity Checker times out.

```
SSSSSSSSSSSSSS
------------------------------
Openshift on Azure end user e2e tests [EndUser][Fake] 
  should create and validate test apps [EndUser][Fake][Apps]
  /go/src/github.com/openshift/openshift-azure/test/e2e/specs/end_user.go:13
STEP: creating test app
time="2019-03-27T13:20:53Z" level=debug msg="creating openshift project for test apps" func="github.com/openshift/openshift-azure/test/e2e/standard.(*SanityChecker).CreateTestApp" file="/go/src/github.com/openshift/openshift-azure/test/e2e/standard/sanitychecker.go:70"
time="2019-03-27T13:20:53Z" level=debug msg="creating stateful test app in e2e-test-xb9oc" func="github.com/openshift/openshift-azure/test/e2e/standard.(*SanityChecker).CreateTestApp" file="/go/src/github.com/openshift/openshift-azure/test/e2e/standard/sanitychecker.go:77"
time="2019-03-27T13:20:53Z" level=debug msg="instantiating django-psql-persistent template" func="github.com/openshift/openshift-azure/test/e2e/standard.(*SanityChecker).createStatefulApp" file="/go/src/github.com/openshift/openshift-azure/test/e2e/standard/testapps.go:25"
time="2019-03-27T13:37:03Z" level=error msg="Get https://bdnumeeyhhtzgxwocegb.eastus.cloudapp.azure.com/apis/template.openshift.io/v1/namespaces/e2e-test-xb9oc/templateinstances/e2e-test-xb9oc: read tcp 172.16.140.19:36488->20.42.32.166:443: read: connection timed out" func="github.com/openshift/openshift-azure/test/e2e/standard.(*SanityChecker).CreateTestApp" file="/go/src/github.com/openshift/openshift-azure/test/e2e/standard/sanitychecker.go:80"

• Failure [970.448 seconds]
Openshift on Azure end user e2e tests [EndUser][Fake]
/go/src/github.com/openshift/openshift-azure/test/e2e/specs/end_user.go:12
  should create and validate test apps [EndUser][Fake][Apps] [It]
  /go/src/github.com/openshift/openshift-azure/test/e2e/specs/end_user.go:13

  Expected
      <[]*standard.TestError | len:1, cap:1>: [
          {
              Bucket: "createStatefulApp",
              Err: {
                  Op: "Get",
                  URL: "https://bdnumeeyhhtzgxwocegb.eastus.cloudapp.azure.com/apis/template.openshift.io/v1/namespaces/e2e-test-xb9oc/templateinstances/e2e-test-xb9oc",
                  Err: {
                      Op: "read",
                      Net: "tcp",
                      Source: {
                          IP: [172, 16, 140, 19],
                          Port: 36488,
                          Zone: "",
                      },
                      Addr: {
                          IP: [20, 42, 32, 166],
                          Port: 443,
                          Zone: "",
                      },
                      Err: {Syscall: "read", Err: 0x6e},
                  },
              },
          },
      ]
  to be empty

  /go/src/github.com/openshift/openshift-azure/test/e2e/specs/end_user.go:17
------------------------------
```
sample artifacts:
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1351/pull-ci-openshift-openshift-azure-master-upgrade-v3.2/48

https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1351/pull-ci-openshift-openshift-azure-master-upgrade-v3.2/46/

/kind test-flake

**Comment by jim-minter** [comment]
possibly another manifestation of #1102

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1377/pull-ci-openshift-openshift-azure-master-e2e-azure/3178 
Azure disk gave this error too. Does not look like it is resource dependent.

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1360/pull-ci-openshift-openshift-azure-master-e2e-azure/3188 
Few in one place

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1360/pull-ci-openshift-openshift-azure-master-e2e-azure/3191

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1357/pull-ci-openshift-openshift-azure-master-e2e-azure/3190

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1377/pull-ci-openshift-openshift-azure-master-upgrade-v3.2/110

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1360/pull-ci-openshift-openshift-azure-master-upgrade-v3.2/119

**Comment by mjudeikis** [comment]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1360/pull-ci-openshift-openshift-azure-master-e2e-azure/3199

**Comment by jim-minter** [comment]
fixed or substantially improved by #1385, closing
